src/coffee/client/modules/todo/todo_list_module.coffee

```diff
@@ -0,0 +1,54 @@
+#
+# Copyright (C) 2015 by Looker Data Services, Inc.
+# All rights reserved.
+#
+
+angular   = require 'angular'
+templates = require '../../templates'
+{EVENT}   = require '../../../constants'
+_         = require '../../../underscore'
+
+############################################################################################################
+
+### ex: TodoList flow
+1. TodoListController listens to TodoListStore and updates its `$scope.todos` when anything changes
+2. TodoListController listens to TodoItemStore fires TodoList's loadAll action when todos are added/removed
+3. Controller fires removeItem/doItem actions based on user events
+
+############################################################################################################
+
+todo = angular.module('todo')
+
+todo.factory 'TodoListStore', (Todo, TodoItemStore, TodoListActions, ReadOnlyView, reflux) ->
```

Comment:
I think you might do better to just remove this store altogether and just add a `getAll` method on your item store.